src/spectrecoff/Spectre/Status.fs

```diff
@@ -10,34 +10,26 @@ type CustomSpinner =
       Look: Look Option }
 
 type StatusOperation = StatusContext -> Task<unit>
+type StatusOperation<'Result> = StatusContext -> Task<'Result>
 
-let start statusText (operation: StatusOperation) =
-    task {
-        return! AnsiConsole
-            .Status()
-            .StartAsync(statusText, operation)
-    }
+type StatusOperationAsync = StatusContext -> Async<unit>
+type StatusOperationAsync<'Result> = StatusContext -> Async<'Result>
 
-let startWithCustomSpinner spinner (operation: StatusOperation) =
-    task {
-        let status = AnsiConsole.Status()
+let private asyncToTask<'T> operation : StatusContext -> Task<'T> =
```

Comment:
Don't you need a `StatusContext<'T>` here? How can the function produce a generic `'T`?

Reply:
Redone